src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java

```diff
@@ -50,8 +51,11 @@ private static double estimateMinorAlleleFraction(final int a, final int r, fina
                     r * (logf + logBias) + a * logOneMinusf - (a + r) * logRefMinorDenominatorTerm;
             return GATKProtectedMathUtils.logSumExp(altMinorLogLikelihood, refMinorLogLikelihood);
         };
-
-        return OptimizationUtils.argmax(objective, AlleleFractionState.MIN_MINOR_FRACTION, AlleleFractionState.MAX_MINOR_FRACTION, initialEstimate);
+        try {
+            return OptimizationUtils.argmax(objective, AlleleFractionState.MIN_MINOR_FRACTION, AlleleFractionState.MAX_MINOR_FRACTION, initialEstimate);
+        }catch (final TooManyArgumentsException ex){
```

Comment:
It's a `TooManyEvaluationsException` from package `org.apache.commons.math3.exception`.